content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -58,9 +58,12 @@ The configuration is provided as an API named
 [`EncryptionConfiguration`](/docs/reference/config-api/apiserver-encryption.v1/). An example configuration is provided below.
 
 {{< caution >}}
-**IMPORTANT:** For high-availability configurations (with two or more control plane nodes), the
-encryption configuration file must be the same! Otherwise, the `kube-apiserver` component cannot
-decrypt data stored in the etcd.
+For cluster configurations with two or more control plane nodes, the encryption configuration
+**must** be identical across each control plane node.
+
+If there is a difference in the encryption provider configuration, this may well mean
```

Comment:
nit: this paragraph could be shortened
Otherwise, the kube-apiserver can't decrypt ...